### ISSUE TITLE
HRC-721 images not showing

### ISSUE THREAD

**Issue by justinasfour04** [harmony-one/chrome-extension-wallet#106]
I'm not sure if this is the issue but it seems like if the nft.uri is missing the following place holder gets displayed
https://github.com/harmony-one/chrome-extension-wallet/blob/cc1e524f5df3caf789ccc2a48a48042079398944/src/popup/pages/Token/HRC721/Gallery.vue#L21

![image](https://user-images.githubusercontent.com/6599551/129507536-f5c83bf0-19c9-464f-90b7-9c64a81d0073.png)

The nfts of Harmonauts are missing the URI but I looked at the 721 protocol and it doesn't seem like it's needed. 
Can someone look into why the NFT images are not appearing on the wallet

**Comment by sophoah** [comment]
@ethlocker are you the correct person to look at that issue ?

**Comment by jasonbakthakumar** [comment]
this issue has not been resolved yet - unable to view images for hrc 721 in browser extension

**Comment by justinasfour04** [comment]
When should we see the changes in the extension? Do we have to manually update?

**Comment by datpenguinclub** [comment]
@justinasfour04 have raised this with on Twitter - https://twitter.com/DatPenguinClub/status/1454099046944149505?s=20

Retweet and remind them to update the release

**Comment by justinasfour04** [comment]
@datpenguinclub is it updated ?

**Comment by datpenguinclub** [comment]
@justinasfour04 yes mate - it's updated - also our pre-sale is live at https://datpenguinclub.com . Come and mint yourself a  🐧  and view them in the browser extension wallet in all their glory. 😉😅

**Comment by datpenguinclub** [comment]
also @justinasfour04 chrome automatically updates all extensions, so I don't think you have to manually update it - but  the version that has the fix is `1.2.5` - the latest one

**Comment by justinasfour04** [comment]
It doesn't seem to be working. I still get the NFT icon

**Comment by datpenguinclub** [comment]
@justinasfour04 could you right click your chrome extension and press "Manage Extension" and make sure that your wallet is on the latest version - `1.2.5`

**Comment by justinasfour04** [comment]
![image](https://user-images.githubusercontent.com/6599551/140859702-3df9d7b1-4e62-47fb-be8c-c803d246255e.png)

**Comment by datpenguinclub** [comment]
@justinasfour04 cool - but with the latest version - with my fix - I'm able to view the NFT along with their attributes 
![wallet_2](https://user-images.githubusercontent.com/93375523/140859806-4f2122ac-9131-4268-aaa0-e1d05a112a2b.png)
![wallet](https://user-images.githubusercontent.com/93375523/140859813-09663a1d-9bba-4e4a-8c70-04b3ed4b51e1.png)

**Comment by justinasfour04** [comment]
I am not

**Comment by justinasfour04** [comment]
I guess I'll fork it and make the change myself and then add to review

**Comment by datpenguinclub** [comment]
@justinasfour04 - could you exit out of the pop-up and open it back up again if it renders - if not it maybe that the asset associated with your NFT is no longer available

**Comment by datpenguinclub** [comment]
@justinasfour04 Well there's no change to be made - this was the change that was preventing the render https://github.com/harmony-one/chrome-extension-wallet/pull/109

**Comment by justinasfour04** [comment]
It works perfectly fine on Metamask, why does this have to be any different. Also I know that all the metadata is there for those NFTs I have

**Comment by justinasfour04** [comment]
![image](https://user-images.githubusercontent.com/6599551/140860277-8e91a001-a0e1-45f8-b445-14042b3bd4ad.png)

**Comment by justinasfour04** [comment]
stuck forever loading

**Comment by datpenguinclub** [comment]
that's on the right path

**Comment by datpenguinclub** [comment]
close the pop-up and open it back up again

**Comment by justinasfour04** [comment]
I am doing that

**Comment by datpenguinclub** [comment]
@justinasfour04 - I reckon the only solution left is uninstalling the chrome extension and re-installing it

**Comment by datpenguinclub** [comment]
@justinasfour04 and to check if it works - come mint a 🐧  at https://datpenguinclub.com 😉

**Comment by justinasfour04** [comment]
I'm not minting a penguin just to see that it works.

**Comment by justinasfour04** [comment]
Removed extension and added it back. Not a single NFT loads

**Comment by datpenguinclub** [comment]
@justinasfour04 well you could try it on the testnet - https://test.datpenguinclub.com - use the faucet https://faucet.pops.one/ to give yourself some ONE on the test network - and you could mint a penguin on test.datpenguinclub.com on the harmony testnet

**Comment by justinasfour04** [comment]
Why are you shilling penguin. I'm telling it's not a good fix

**Comment by datpenguinclub** [comment]
@justinasfour04 I'm not shilling you my penguin - it costs you $0 to test it on the test network.

**Comment by justinasfour04** [comment]
How about you test it with other nfts
like pixel pirates or the frey or harmonauts

**Comment by datpenguinclub** [comment]
@justinasfour04 if you don't think it's a proper fix - you're welcome to give a better fix. I was just trying to tell you that my fix seems to fix not being able to view NFTs (attached screenshot above)..

Are any of these other projects like pixel pirates / freys and the harmonauts on the harmony testnet ?? if yes what's their URL ?? More than happy to test them out

**Comment by justinasfour04** [comment]
https://explorer.testnet.harmony.one/address/0xa5b261935bf555df9cc29b7801ddd00ce1800ff1

**Comment by datpenguinclub** [comment]
@justinasfour04 I'm just trying to tell you that this issue has been solved - and I have screenshots both on the testnet and mainnet - I'm not trying to shill you anything - not an opinion - a fact 

That's an explorer page - I can't mint anything on there - I need URLs to the projects test environments running on the harmony testnet - not the explorer page.

**Comment by datpenguinclub** [comment]
Do you have the URL to testnet version of harmonauts / or freys / or any other Dapps you mentioned ?? I mentioned mine - cause I have a test version of my Dapp running on the harmony testnet -  so you can test it without any real $$.

Cause we'd like to have end to end integration and best UX when users do use our Dapp. So I'm not shilling you anything, in fact no one is asking me to any of this - I'm just here trying to help. I'm signing off for now 😶🤷‍♂️

**Comment by justinasfour04** [comment]
Issue fixed in the following https://github.com/harmony-one/chrome-extension-wallet/pull/111
![image](https://user-images.githubusercontent.com/6599551/140871584-035409ca-10e4-4a86-aa7a-6c4ace91ff2b.png)
![image](https://user-images.githubusercontent.com/6599551/140871611-45d5106a-f258-41a7-b57b-d99f54791a83.png)

**Comment by justinasfour04** [comment]
@datpenguinclub @givp

**Comment by justinasfour04** [comment]
Sorry to bother you, but can someone have a look at this?
